modules/ROOT/pages/pages-development.adoc

```diff
@@ -60,25 +60,6 @@ resources=[GET|identity/user, GET|identity/personalcontactdata, GET|identity/pro
 
 When a user is given access to this page because they are in a profile that contains it or that is mapped to an application that contains the page, then this user is granted, upon login, the permissions associated to these resources (see xref:rest-api-authorization.adoc[REST API authorizations] for more details).
 
-=== Custom page examples
-
-Two custom page examples are available in Bonita Portal. Both examples show how to:
-
-* Get the Bonita CSS
-* Write simple HTML code
-* Get session information, including the locale
-* Get resources from the custom page definition
-* Call the Engine Java APIs (in the Groovy example) or the Web REST APIs (in the.md example)
-* Write a clickable link to an external page
-* Write a clickable link to a portal page
-* Write locale-specific messages
-
-*Groovy example page* defines a custom page using Groovy.
-*HTML example page* defines a custom page using only HTML.
-In practice, you will probably use a combination of Groovy and HTML to create your custom pages.
-
-To view an example page, map the page to an xref:applications.adoc[application] so that you can view it in Bonita Portal. To view the source of an example, xref:resource-management.adoc]#export[export the custom page].
-
 === Reuse Bonita Portal content
```

Comment:
to be deleted?

Reply:
Part of [RUNTIME-525](https://bonitasoft.atlassian.net/browse/RUNTIME-525)

Reply:
It will be deleted when working on that ticket

Reply:
got it! ;)